x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/mock_alert_result_list.ts

```diff
@@ -47,13 +52,17 @@ export const mockAlertResultList: (options?: {
         },
       },
       thread: {},
+      prev: null,
+      next: null,
     });
   }
   const mock: AlertResultList = {
     alerts,
     total,
     request_page_size: requestPageSize,
     request_page_index: requestPageIndex,
+    next: '/api/endpoint/alerts?after=1542341895000&after=2f1c0928-3876-4e11-acbb-9199257c7b1c',
```

Comment:
should this have two of the same url params?

Reply:
@alexk307 Yep, that's how you represent arrays in GET requests.